### ISSUE TITLE
Registration Issues

### ISSUE THREAD

**Issue by RobertGleaden** [Phalcode/crackpipe-app#12]
When using the UI to register for an account I get a 
"Each field must be filled"

![image](https://github.com/Phalcode/crackpipe-app/assets/3193292/51e57d11-5955-4db1-b212-dfa74af2722c)

**Comment by Alfagun74** [comment]
The fix is coming in V1.0.1 and is on its way to the Microsoft Store right now.

**Comment by Alfagun74** [comment]
In a nutshell: We opted for the Windows Store since it's the most budget-friendly option that gains Microsoft Windows' trust without bombarding users with countless warnings about the safety of our unfamiliar .exe file. Instead, users can smoothly install the app without any hassle. Moreover, the store offers a user-friendly update process. Although Microsoft reviews each update, it assures users of the authenticity of our software and protects against potential threats. Obtaining a code signing certificate is beyond our budget (costs ~€900 for 3 years).

**Comment by Alfagun74** [comment]
https://github.com/Phalcode/crackpipe-app/issues/14

**Comment by Eyeanman** [comment]
Hey, post update, I now get this error:
![image](https://github.com/Phalcode/crackpipe-app/assets/45572030/14f45eb9-b268-43ee-a7ae-b235b942263d)

Version showing in Store:
![image](https://github.com/Phalcode/crackpipe-app/assets/45572030/cafd835e-ef99-4264-8e09-2282b996e854)

**Comment by Alfagun74** [comment]
Yeah you are right, just select another image for now, well think about making those fields optional.

**Comment by Eyeanman** [comment]
> Yeah you are right, just select another image for now, well think about making those fields optional.

Can you provide an example URL that is known to work please? Ive tried a number of different ones (including selfhosting) but no luck :(

**Comment by Alfagun74** [comment]
Don't worry. Images will be optional for registration in a coming update anyway, also I will need to improve the error handling on the downloader and maybe even use a different one altogether.
